Mage/src/main/java/mage/abilities/condition/common/ManaWasSpentCondition.java

```diff
@@ -19,31 +19,48 @@ public enum ManaWasSpentCondition implements Condition {
     BLUE(ColoredManaSymbol.U),
     BLACK(ColoredManaSymbol.B),
     RED(ColoredManaSymbol.R),
-    GREEN(ColoredManaSymbol.G);
+    GREEN(ColoredManaSymbol.G),
+    COLORLESS(null, true);
```

Comment:
You might want to rebase. I've done similar change already.

https://github.com/magefree/mage/commit/f30ec10846f5e0962f2faccdc55bc6878eb01fa9#diff-37ab843521f2456b64927c2baeae3c7e9d0439d947bb3b04742e0b3f7ade81d9